phpstorm/CheckBuildUpdate.ps1

```diff
@@ -44,6 +53,8 @@ Write-Host "Update nuspec"
 $nuspec_template.package.metadata.version = $newVersion
 if (Test-Path "$directory\phpstorm.nuspec") { Remove-Item "$directory\phpstorm.nuspec" }
 $nuspec_template.save("$directory\phpstorm.nuspec")
+Write-Host "Update nuspec <releaseNotes> with new URL"
+(Get-Content "$directory\phpstorm.nuspec") -replace('{{release_url}}', $release_url) | Set-Content "$directory\phpstorm.nuspec"
```

Comment:
Please do the replacement in the template object like with the version number 2 lines above.
String replacement should be avoided whenever possible. In this case the NuSpec file is already plain XML and can thus be treated like a first-class-citizen in the PowerShell world.
String replacement is only neccessary in the powershell files since we can't address specific parts of these files directly.